sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAdministrationServiceVersion.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.administration;
+
+import com.azure.core.util.ServiceVersion;
+
+/**
+ * The versions of Azure Key Vault Administration service supported by this client library.
+ */
+public enum KeyVaultAdministrationServiceVersion implements ServiceVersion {
+    V7_0("7.0"),
+    V7_1("7.1"),
```

Comment:
Are these valid version strings? I believe the service doesn't currently enforce any, but will that always be true?

Reply:
For MHSM, no. They are new with 7.2-preview. Even if/when AKV supports any of these, it'll start at whatever version. At this point, MHSM and KV need to be in lock-step for api-version. /cc @jlichwa @amitbapat

Reply:
I think I should remove the older versions since they don't support KeyVaultAdministration, I think I added them because I based this class off of what we have in other KV SDKs. What do you think?

Reply:
Yes, I would remove these values for the Administration package.